lib/matplotlib/backend_bases.py

```diff
@@ -3237,36 +3237,36 @@ def _tool_toggled_cbk(self, event):
 
         This only gets used for toggled tools
         """
-        self.toggle_toolitem(event.tool.name, event.tool.toggled)
+        self.toggle_toolitem(event.tool_name, event.tool.toggled)
 
-    def add_tool(self, tool, group, position=-1):
+    def add_tool(self, tool_name, group, position=-1):
         """
         Adds a tool to this container
 
         Parameters
         ----------
-        tool : tool_like
-            The tool to add, see `ToolManager.get_tool`.
+        tool_name : str
```

Comment:
This is an API change, isn't it?

Reply:
As are the added args below?  Not sure if they are a problem, but prob needs an Api change note